docs/source/guide/tutorials/base/demo.rst

```diff
@@ -57,57 +75,10 @@ The output will look like this:
 
 After that, we can use this ``/dev/video0`` as a camera ID for ``--input``.
 
-5. We can also change ``config.json`` that specifies the confidence threshold and color for each class visualization, but any changes should be made with caution. 
```

Comment:
Can we change some parameters using OTX CLI inside OTX?

Reply:
As I can see, no